week5/5.1/I have no vinaigrette.py

```diff
@@ -0,0 +1,48 @@
+import time
+import random
+from datetime import datetime
+"""
+I have no vinaigrette!.
+By: Eliad Karni.
+
+The Project generates a date between two received dates.
+If the generated day is not a SHOPPING_DAY, it prints NO_VINAIGRETTE_DATE_OUTPUT message.
+
+used websites help:
+shopping day check -> https://www.delftstack.com/howto/python/python-datetime-day-of-week/
+                      https://www.geeksforgeeks.org/python-datetime-strptime-function/
+generate date -> https://stackoverflow.com/questions/553303/generate-a-random-date-between-two-other-dates
+"""
+
+TIME_FORMAT = '%Y-%m-%d'
+SHOPPING_DAY = 0  # monday: 0, sunday: 6
+NO_VINAIGRETTE_DATE_OUTPUT = "I have no vinaigrette!"
```

Comment:
I wouldnt add it as a const if it is used once, but thats only me

Reply:
I think it makes the code more readable and changeable...
I'll change it anyway...